README.md

```diff
@@ -1,3 +1,30 @@
 # nodule-openapi
 
-Opinionated OpenAPI (Swagger) Client
+Opinionated OpenAPI 2.0 (Swagger) Client
+
+
+## Conventions
+
+ -  Client code should be lightweight (e.g. few dependencies) and initialize quickly (read: synchronously)
+
+    As such, OpenAPI specs are expected to be available locally (e.g. commited to version control) and
+    loaded statically (i.e. without requiring promise resolution).
+
+ -  Client interfaces should use fluent JavaScript.
+
+    As of this writing, OpenAPI operations are exposed as functions named after `operationId`, scoped to an
+    an object named after the operation's first `tag`. (Both operation fields are thus mandatory).
+
+ -  HTTP requests are assumed to use camelCase for body parameters and snake_case for path and query string
+    parameters; parameter names should be automatically converted from camelCase accordingly.
+
+
+## Usage
+
+In pseudo-code:
+
+    const OpenAPI = require('./lib').default;
```

Comment:
Obviously the `./lib` can change here... and the style could be babelized if we wanted to be less friendly.

The request below will actually fail on a 404 because I think the petstore.io site is no longer well-maintained. More/better examples would obviously be useful.